README.md

```diff
@@ -62,41 +66,50 @@ I write my personal thoughts on :gear: Tech & :seedling: Life in my :writing_han
 * I feel more focused with a hoodie puts on
 * I wake up at 4:40 followed with a cold shower
 * I'm a huge fan of Iron Man and Hulk
-* I'm a big fan of *Universe of Metro 2033* - mostly books and books "Daemon" and "Freedom" written by Daniel Suarez
+* I really enjoyed *Universe of Metro 2033* books series; also a big fan of two books: "Daemon" and "Freedom" written by Daniel Suarez.
 * I use ![Todoist](https://img.shields.io/badge/-Todoist-E44332?logo=Todoist&logoColor=fff) everyday
 
 ## :hourglass_flowing_sand: Things I don't know, I want to master or just improve (randomly ordered)
 
-* Software Architecture, microservices and distributed systems as a whole
+* Software Architecture, microservices, distributed systems and all the fancy keywords
+* Dive into Domain-driven design
+* Software Design Patterns
 * Software Craftsmanship
 * Learn teaching and mentoring
 * code search and read with **grep** and **find** from command line (only ack-grep right now)
-* best practices API design and security
+* best practices related with API design and its security
 * Golang
 * Vuetify & TailwindCSS
-* Docker ecosystem & Kubernetes
-* Building and managing CI/CD: Travis CI & GitHub Actions
+* deeper understanding of Docker ecosystem with Kubernetes
+* Building and managing CI/CD pipes: Travis CI & GitHub Actions.
```

Comment:
you have added dot on end of this sentence but you are not having dots nor commas on others. Be consistent ;)

Reply:
Nice catch, thanks.